src/Maestro/tests/Scenarios/common.ps1

```diff
@@ -164,10 +164,8 @@ function Darc-Command() {
 }
 
 function Darc-Command-Impl($darcParams) {
-    $baseDarcCommand = "& $darcTool $darcParams" 
     Write-Host "Running 'darc $darcParams $darcAuthParams'"
-    $darcCommand = "`$commandOutput = $baseDarcCommand $darcAuthParams; if (`$LASTEXITCODE -ne 0) { Write-Host `${commandOutput};throw `"Darc command exited with exit code: `$LASTEXITCODE`" } else { `$commandOutput }"
-    Invoke-Expression $darcCommand
+    $commandOutput = & $darcTool $darcParams $darcAuthParams; if ($LASTEXITCODE -ne 0) { Write-Host ${commandOutput};throw "Darc command exited with exit code: $LASTEXITCODE" } else { $commandOutput }
```

Comment:
`
Verb 'add-channel --name '658958354' --classification 'test'' is not recognized.
`
The `"` from $darcParams seem to be getting passed along, making it so it looks like the command is the entire string.

Reply:
I can't remember why I wrote it like that originally, but there may have been a fair bit of trial and error there related to quoting semantics and such.  This needs some local testing prior to check-in.

Reply:
Yep, I'll be making sure to figure out this quote stuff.